### PR TITLE
feat: connected groupchats and createGroupChat

### PR DESCRIPTION
**Pull request by hits-art01**

### CODE REVIEW

File: src/components/CurrentChat/CurrentChat.js [hunk 1]
```diff
@@ -40,7 +42,10 @@ const CurrentChat = () => {
             </div>
             <div
               className={style.chat_header_settings}
-              onClick={() => setIsChatSettingsVisible(!isChatSettingsVisible)}
+              onClick={() => {
+                // navigate(`/chats/${current.name}/about`);
```
Comment (Andrii-Pinchuk) [inline src/components/CurrentChat/CurrentChat.js]: Why do you need a commented code here?

File: src/pages/ChatMainPage/ChatMainPage.js [hunk 2]
```diff
@@ -41,19 +41,34 @@ const ChatMainPage = () => {
...
   };
 
+  // useEffect(() => {
+  //   if (pathname === "/chats/search") {
+  //     setIsSearch(true);
+  //   } else {
+  //     setIsSearch(isSearch);
+  //   }
+  // }, [pathname]);
+
```
Comment (Andrii-Pinchuk) [inline src/pages/ChatMainPage/ChatMainPage.js]: remove unnecessary commented code, please

File: src/pages/ChatMainPage/chat-main-page.module.css [hunk 3]
```diff
@@ -125,9 +125,21 @@ select:focus {
   border: 1px solid var(--secondary-background-illustration, #d7d2e6);
   padding: 10px;
 }
+.aside_input_wrap_mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 350px;
+  /* margin-bottom: 20px; */
```
Comment (Andrii-Pinchuk) [inline src/pages/ChatMainPage/chat-main-page.module.css]: remove

File: src/components/CurrentChat/current-chat.module.css [hunk 4]
```diff
@@ -153,7 +153,7 @@
   position: absolute;
   background: #fff;
```
Comment (Andrii-Pinchuk) [inline src/components/CurrentChat/current-chat.module.css:154]: Use only variables for colors, please. This applies to the entire project.

File: src/pages/ChatMainPage/MobileVersion.js [hunk 5]
```diff
@@ -162,7 +163,13 @@ const MobileVersion = () => {
             <div className={style.header_logo}>
               <img src={Logo} />
             </div>
-            <div className={style.header_search}>
+            <div
+              className={style.header_search}
+              onClick={() => {
+                navigate("/chats/search");
+                dispatch(getGroups());
+              }}
```
Comment (Andrii-Pinchuk) [inline src/pages/ChatMainPage/MobileVersion.js]: It's better to write a separate function or method and use it here in onClick. Especially if it's length is more than 1 row.

File: src/pages/ChatMainPage/chat-main-page.module.css [hunk 6]
```diff
@@ -125,9 +125,21 @@ select:focus {
...
+
+  border-bottom: 1px solid var(--secondary-background-illustration, #d7d2e6);
```
Comment (Andrii-Pinchuk) [inline src/pages/ChatMainPage/chat-main-page.module.css]: Why you can't use only var here and move #d7d2e6 to var initialization?

File: src/components/CurrentChat/current-chat.module.css [hunk 7]
```diff
@@ -153,7 +153,7 @@
...
 
-  top: 59px;
+  top: 60px;
   left: -321px;
   height: 800px;
   width: 370px;
```
Comment (Andrii-Pinchuk) [inline src/components/CurrentChat/current-chat.module.css]: I do not recommend using absolute values for height and width, except for images (png, etc) and places where the width or height is not determined by the content. If there is content inside the block and there is a certain distance from the content to the edges, then you can set paddings. This applies to the entire project. In the project code, in addition to this, I saw many places where similar bones are used. This leads to bugs in adaptability and difficulties in further code support.

**Review by Andrii-Pinchuk — CHANGES_REQUESTED** [review]

File: src/pages/ChatMainPage/chat-main-page.module.css (hunk 6 above)
Comment (hits-art01) [reply src/pages/ChatMainPage/chat-main-page.module.css]: Just copied from figma

**Review by Andrii-Pinchuk — APPROVED** [review]

**Pull request merged**